src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -1984,7 +1984,7 @@ public void resetChangeMonitor() {
     }
 
     public void updateTimeStamp() {
-        changeMonitor.ifPresent(DatabaseChangeMonitor::markAsSaved);
+        changeMonitor.ifPresent(DatabaseChangeMonitor::updateTimestampAndFileSize);
```

Comment:
I would actually prefer to rename `updateTimeStamp()` to `markAsSaved`. That the `DatabaseChangeMonitor` uses the timestamp and file size to keep track of the last saved state, should be an implementation detail that is hidden to the outside world.

Reply:
Alright, I changed the name back to what it was before.